### ISSUE TITLE
trying to mount a file that for some reason fails to mount shows no error

### ISSUE THREAD

**Issue by Masamune3210** [x1nixmzeng/xbox-iso-vfs#2]
Attempting to mount a image that has a issue keeping it from mounting doesn't print anything to the user to say as such. Currently, it just drops back to the prompt after a bit of loading

**Comment by x1nixmzeng** [comment]
Can you expand what you mean by "issue keeping it from mounting" - do you mean it's on a removal drive?

**Comment by Masamune3210** [comment]
No, one iso from redump refused to process correctly no matter what I do with any tool due to Invalid TOC entries. At least that's what most tools complain of

**Comment by Masamune3210** [comment]
It's Hunter the Reckoning (USA) iirc. Will check when I get home and message if not

**Comment by x1nixmzeng** [comment]
I've pushed a branch that aims to fix this. 

Could you download the build from this link and let me know if this fixes your issue: https://ci.appveyor.com/project/x1nixmzeng/xbox-iso-vfs/builds/37124513/artifacts

If not, it's worth checking your file is not corrupt - the redump site shows the expected CRC32/MD5/SHA-1 hashes:
http://redump.org/disc/50450/

**Comment by Masamune3210** [comment]
![image](https://user-images.githubusercontent.com/1053504/103731884-d961cf80-4fab-11eb-80b5-e01a11e1d0f8.png)
Image mounts seemingly perfectly now! Is there any risk of data corruption or missing data due to the invalid TOC?

**Comment by x1nixmzeng** [comment]
Thanks for confirming 👍 

Regarding any missing data, I found [this comment](https://github.com/JayFoxRox/extract-xiso/blob/624847af8d73ec397575c8bad531223a048821b4/extract.py#L41-L64) that explains how it might occur, and did a similar check for my fix. I took a quick look at another tool (XDVDMulleter) and it will just fail if it encounters a file entry from a padded block:

```c#
      if (xfsInputFile.StartSector == -1)
        throw new SystemException("Invalid TOC entry");
```

I'll do a bit more investigation to see the sort of coverage skipping these sectors gets. In terms of missing data - empty directories may not be created.
